client/state/index.js

```diff
@@ -58,6 +58,7 @@ let middleware = [ thunkMiddleware ];
 if ( typeof window === 'object' ) {
 	middleware = [
 		...middleware,
+		require( './analytics/middleware.js' ).analyticsMiddleware,
 		require( './themes/middlewares.js' ).analyticsMiddleware
```

Comment:
It's hard not to ask about about the middleware put here by the themes team and why we are not using it.